packages/app/src/sessions.ts

```diff
@@ -147,16 +154,18 @@ export function useSessions(
           {
             select: [
               {
-                valueExpression: `DISTINCT ${sessionSource.resourceAttributesExpression}['rum.sessionId']`,
+                valueExpression: `DISTINCT ${SESSION_TABLE_EXPRESSIONS.resourceAttributesExpression}['rum.sessionId']`,
```

Comment:
@wrn14897 so it looks like the very fields we are looking to remove for the session form, we are using in these queries. Did we need these afterall?